main/reentrancy.c

```diff
@@ -187,11 +187,13 @@ PHPAPI char *php_ctime_r(const time_t *clock, char *buf)
 	local_lock(CTIME_R);
 
 	tmp = ctime(clock);
-	strcpy(buf, tmp);
+	if (tmp) {
+		strcpy(buf, tmp);
+	}
```

Comment:
Same for asctime?

Reply:
Not necessary according to the [docs](https://docs.microsoft.com/en-us/cpp/c-runtime-library/reference/asctime-wasctime?view=vs-2019), but nonetheless a good idea.